package.json

```diff
@@ -76,10 +73,9 @@
     "require-dir": "^1.1.0",
     "signale": "^1.3.0",
     "standard-version": "4.2.0",
+    "start-server-and-test": "^1.7.13",
```

Comment:
This is the same package we use in `nui-widget` to run puppeteer tests in Circle.